mobileportfolio.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Portfolio</title>
+    <link rel="stylesheet" href="mobileportfolio.css">
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
+                
+</head>
+<body>
+    <main class="bg-shapes">
+       <header class="logo-nav">
+            <nav class="navbar">
+                <div class="my-logo">My Logo</div>
+                <a href="#" class="toggle-button">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </a>
+                <div class="hamburger-menu">
+                    <ul>
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        
+        <section>
+            <div class="headline">
+                <h2>I'm Mariane <br> Glad to see you &#33;</h2>
```

Comment:
- [ ] [OPTIONAL] You have done well on implementing this change requested by the previous reviewer but you can optionally change the div tag that has the class ```headline``` to a section tag. This will make your code cleaner.

Reply:
@amon-cofie Well noted. I will take this into account. so a code can have several section tags to differentiate a h tag and then use div for the content?